pkg/test/constants/presubmitconstants.go

```diff
@@ -42,6 +42,7 @@ var (
 		"cloudbuild":           {"cloudbuildtrigger"},
 		"cloudfunctions":       {"httpsfunction"},
 		"cloudidentity":        {"addexpirydatecloudidentitymembership"},
+		"cloudids":             {"idsendpoint"},
```

Comment:
If `CloudIDS` is the name we want to stick with, then the folder of the test case should be the full resource name as well, i.e. `cloudidsendpoint`. Instructions: https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/README.NewResourceFromTerraform.md#add-necessary-yaml-files-to-testdata

I'm actually in support of updating the service name to `IDS` because this is the host name of the API: https://cloud.google.com/intrusion-detection-system/docs/configuring-ids#create_a_endpoint

Reply:
Just bridging some online conversations. We may choose to rename this down the road if that is possible at all (since renaming the CRDs might not be possible), but going with the autogenerated `cloudids` for now.